.github/ISSUE_TEMPLATE/confiq.yml

```
@@ -0,0 +1,30 @@
+name: meta/repository Report
+description: for requesting changes and additions to the repo itself.
+title: "meta/ repository issue"
+labels: ["updates"]
```

We don't have an update label, but we do have a label for 📇 Meta/Repository
```suggestion
labels: ["📇 Meta/Repository"]
```